### PR TITLE
Ismn new data (issue 266)

### PR DESCRIPTION
**Pull request by pstradio**
This PR is for reading the new ISMN dataset version, downloaded on 2021.01.31. 

- The new dataset has already been downloaded on the server and a test dataset has been added on a PR to the testdata repository.
- The new dataset readme.txt file has been added under `validator/static/files/`
- The `FIL_ISMN_NETWORKS` filter in filters.json has been updated to include all networks by default (it can be changed in the validation page)
- `testdata/` has been included in .gitignore

let me please know whether any of these changes should be amended

### CODE REVIEW

**Comment by pstradio** [comment]
I think the test fails because the PR on the testdata repository has not been merged yet, I hadn't thought about it. I'll try again after that has happened

**Comment by sheenaze** [comment]
I merged test data repository, but for some reason I'm not able to fetch the data :( I'll check it tomorrow. Also, when I fetched your branch and run server, there is no version mini for ISMN.

**Comment by sheenaze** [comment]
Sorry, my bad, everything looks ok :) I have to check why the tests fail.

**Comment by pstradio** [comment]
This is strange, I tried connecting my local repository data to the testdata repository and all the tests are passed, My guess is that it has to do with the creation of the python_metadata folder for some reasons. This has been quite problematic in some cases but I need to figure out why; I'll try to fix this :)

**Comment by pstradio** [comment]
It's funny because when I click on the link to the testdata repo from the code it links to a version previous to my merge (with the new directory). Is there anything that needs to be done to update the testdata link @sheenaze ?

**Comment by sheenaze** [comment]
@wpreimes will it help if I remove metadata from qa4sm-testdata repository?

**Comment by sheenaze** [comment]
Which link?

**Comment by pstradio** [comment]
> Which link?
This link:
https://github.com/awst-austria/qa4sm-testdata/tree/51fd749cd0d94fd73455a711407b81819626f1ad

I think the problem is with the `submodule` but I don't know how to update it

**Comment by sheenaze** [comment]
Where this link comes from?

**Comment by pstradio** [comment]
> Where this link comes from?

This is from qa4sm/testdata in this git repository code. I updated the `submodule` now, let's see if this works

**Comment by sheenaze** [comment]
With the changes you applied I'm not able to load testdata at all :(

**Comment by sheenaze** [comment]
bravo :)

**Comment by pstradio** [comment]
> bravo :)

Before the merge, should I add/change the test for the new dataset?

**Comment by sheenaze** [comment]
Yes, I think it is a good idea.

File: validator/fixtures/networks.json [hunk 1]
```diff
@@ -766,19 +823,21 @@
...
             "versions": [
+                30,
                 18
             ]
         }
     },
     {
         "model": "validator.ismnnetworks",
-        "pk": 60,
+        " ": 60,
```
Comment (sheenaze) [inline validator/fixtures/networks.json:833]: why "pk" is removed here?

**Comment by sheenaze** [comment]
Ok, now it looks like everything is ok. You can merge it and I'll do the deployment to the test instance.

**Review by sheenaze — APPROVED** [review]

**Pull request merged**